### ISSUE TITLE
Website Accessibility-WCAG 2.4.4,WCAG 4.1.2: Ensures links have discernible text (.navbar-brand)

### ISSUE THREAD

**Issue by A11Ybot-OP** [bazelbuild/bazel#14492]
#### Issue
`Ensures links have discernible text` ([link-name](https://accessibilityinsights.io/info-examples/web/link-name))
#### Target application
[Bazel - a fast, scalable, multi-language and extensible build system" - Bazel](https://bazel.build/)
#### Element path
.navbar-brand
#### Snippet
`<a class="navbar-brand" href="/">
            <img class="navbar-logo" src="/images/bazel-navbar.svg">
          </a>`
#### How to fix
    Fix all of the following:
      Element is in tab order and does not have accessible text
    
    Fix any of the following:
      Element does not have text that is visible to screen readers
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute
#### Environment
Microsoft Edge version 96.0.1054.62

**Comment by fweikert** [comment]
We've migrated to a new site at https://bazel.build that fixes several accessibility issues.
